itest/godot/ManualFfiTests.gd

```diff
@@ -24,4 +25,18 @@ func test_to_string() -> bool:
 
 	print("to_string: ", s)
 	print("to_string: ", ffi)
-	return true
\ No newline at end of file
+	return true
+	
+func test_export() -> bool:
+	var obj = HasProperty.new()
```

Comment:
Since this is manually managed, you need to call `obj.free()` in the end. At the moment, the integration test exits [with a memory leak warning](https://github.com/godot-rust/gdextension/actions/runs/3703451505/jobs/6274919243#step:3:844). Yes, I should check that in CI 😉 

Also, could you add some empty lines to group related GDScript statements, similar to other functions? It looks quite cluttered at the moment.

Reply:
Done.